src/player.cpp

```diff
@@ -5226,7 +5226,7 @@ void player::suffer()
         }
     }
 
-    if( has_trait( trait_SHARKTEETH ) && one_in( 14400 ) ) {
+    if( has_trait( trait_SHARKTEETH ) && one_in( to_turns<int>( 24_hours ) ) ) { //originaly 14400
```

Comment:
This comment seems uneccesary.

Reply:
It is unnecessary, I had those around so I could see the original values were. I must have missed one.